core/app/models/spree/order.rb

```diff
@@ -620,13 +618,6 @@ def ensure_line_items_present
       end
     end
 
-    def has_available_shipment
-      return unless has_step?("delivery")
-      return unless has_step?(:address) && address?
-      return unless ship_address && ship_address.valid?
-      # errors.add(:base, :no_shipping_methods_available) if available_shipping_methods.empty?
```

Comment:
@tanmay3011 - Please add a deprecation warning for this method, rather than removing it.